testing/it/core-wcm-components-it-content/src/content/META-INF/vault/filter.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2021 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
+<workspaceFilter version="1.0">
+    <!-- Sample content -->
+    <filter root="/conf/core-components"/>
+    <filter root="/content/core-components"/>
+    <filter root="/content/campaigns/core_component_brand"/>
+    <filter root="/content/dam/core-components"/>
+    <filter root="/content/cq:tags/core-components"/>
+    <filter root="/content/experience-fragments/core-components-test"/>
+    <filter root="/home/users/core-component"/>
+    <filter root="/etc/blueprints/corecomp_blueprint"/>
+    <!-- Hobbes IT folder -->
+    <filter root="/apps/core/wcm/tests"/>
+    <filter root="/apps/msm">
+        <include pattern="/apps/msm/corecomp_blueprint(/.*)?"/>
```

Comment:
Are the blueprints used? Perhaps we can drop them

Reply:
Not required now. For 6.5 having this configuration is necessary to have Blueprint Tab in properties page. But in 6.6 the dependency was removed since this configuration exists in /libs or /apps and it can not be modified on the run. This dependency removal change need to be backported in 6.5 as well.
I have marked the test failing on 6.5 for now. We can enable the test once that fix is released.

Reply:
@vladbailescu I think we can remove the /apps folder completely. Is there any dependency on that path for http tests you are aware of?